### ISSUE TITLE
Ability to scrollTo without animation

### ISSUE THREAD

**Issue by nmikhailov** [MurhafSousli/ngx-scrollbar#29]
### Bug Report or Feature Request (mark with an `x`)
```
- [ x] bug report -> please search issues before submitting
- [ ] feature request
- [ ] question
```

### Versions
Angular: 5.2.10

### Repro steps

Stackbliz: https://stackblitz.com/edit/ngx-scrollbar-eccrcv

1. scrollYTo called with low durations(<10) works incorrectly. 
2. scrollYTo doesn't work if called inside parent's ngOnInit hook. It produces error:
ERROR TypeError: Cannot read property 'scrollTop' of undefined

### Desired functionality
1. I want to be able to scroll instanteniously, without animations, ie scrollYTo(yPos, 0)
2. I want to be able to scroll inside parent's ngOnInit. Component should remember desired scrollPosition and apply it after it is initialized.

**Comment by MurhafSousli** [comment]
Yupp, the minimum delay should be not less not a 100ms

https://github.com/MurhafSousli/ngx-scrollbar/blob/9a5e9c3d5012d55c487e812c43c5b40f46d50596/projects/ngx-scrollbar/src/lib/scrollbar.component.ts#L152-L169

**Comment by nmikhailov** [comment]
So is this WONTFIX or what?

**Comment by MurhafSousli** [comment]
10ms is a very tiny delay. if you have another solution, share it

**Comment by nmikhailov** [comment]
I am not that interested in small delays, though.
As I have pointed in 'Desired functionality' I want to scroll without any delays and animations at all.

I would propose to do something like
if (duration === 0 /* or maybe even < 10 */) {
   this.renderer.setProperty(this.view, 'scrollTop', ...)
} else {....}

**Comment by MurhafSousli** [comment]
Fixed in v2.0.4
- Without animation: `scrollYTo(y)`
- With animation: `scrollYTo(y, duration)`
